nipoppy/workflows/dataset_status.py

```diff
@@ -62,7 +62,25 @@ def run_main(self):
             [status_col_dict["manifest"]] + curation_cols + processing_cols
         ]
 
-        self.logger.debug(status_df)
+        self.logger.debug(f"curation columns: {curation_cols}")
+        self.logger.debug(f"processing columns: {processing_cols}")
+
+        # check if the number of bids participants match the manifest
+        # note: not checking participant IDs, just the number of participants
+        if curation_cols and status_df[self.curation_status_table.col_in_bids].equals(
+            status_df["in_manifest"]
+        ):
+            self.logger.info(
+                "The number of BIDS participants match the manifest; hiding "
+                "counts for pre- and post-reorg stages."
+            )
```

Comment:
What do you think of this instead?

```suggestion
            self.logger.info(
                "All participants in the manifest have BIDS data; hiding "
                "counts for pre- and post-reorg stages."
            )
```

Reply:
Technically, we are only checking the counts and not exact IDs. I know the assumption is that the identical count is highly unlikely if they are not the same participants, but just in some rare future case if someone needed to debug this scenario, I didn't want to have a misleading log message.